Dolapo47/Auto-Mart
# create test case for post and get route

#### What does this PR do?
- A summary of what this task is about
this pr writes the test for car routes get all, get available get available within price range, get single car
#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
import mocha and chai-http to the test environment
import app 
#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
run npm test on the terminal

#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[#166980708](https://www.pivotaltracker.com/story/show/story_id)`)
